.circleci/config.yml

```diff
@@ -70,6 +70,19 @@ workflows:
             - build_docker
             - helm_lint
           helm_timeout: 5m
+      - hmpps/deploy_env:
+          name: deploy_preprod
+          env: "preprod"
+          jira_update: true
+          jira_env_type: staging
+          pipeline_id: <<pipeline.id>>
+          pipeline_number: <<pipeline.number>>
+          context:
+            - hmpps-common-vars
+            - hmpps-template-kotlin-preprod
```

Comment:
This may be correct, but just checking if this should be referencing the template?

Reply:
Thanks for spotting this @rgforsyth - I've removed it as it isn't there for dev chunk so I assume not required,.